src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :hasDraftPlanOfArrangement="hasDraftPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
+        @emitCourtNumber="setFileNumber($event)"
+        @emitPoa="setHasPlanOfArrangement($event)"
+        @emitValid="setValidCourtOrder($event)"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { CourtOrderPoa as CourtOrderPoaShared } from '@bcrs-shared-components/court-order-poa'
+import { ActionBindingIF, FlagsReviewCertifyIF } from '@/interfaces/'
+
+@Component({
+  components: {
+    CourtOrderPoaShared
+  }
+})
+export default class CourtOrderPoa extends Vue {
+  /** Prop to provide section number. */
+  @Prop({ default: '' }) readonly sectionNumber: string
+
+  /** Store getters */
+  @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
+  @Getter getAppValidate!: boolean
+  @Getter getHasPlanOfArrangement!: boolean
+
+  /** Global actions */
+  @Action setValidCourtOrder!: ActionBindingIF
+  @Action setHasPlanOfArrangement!: ActionBindingIF
+  @Action invalidCourtOrder!: ActionBindingIF
+
+  /** LocalProperties */
+  private courtOrderNumber: string = null
+  private planOfArrangement: boolean = false
+
+  /** Flags to enable validation */
+  private setFileNumber (courtOrderNumber: string): void {
```

Comment:
This probably overrides the the `setFileNumber` action that was imported when this component extended file-template-mixin.ts.

If a mixin used by this component already includes a store getter or setter (or any variable or method, really) then you should get a warning when you try to re-declare it here.

In this case, the action from the mixin doesn't have to be redeclared here, and re-declaring it probably overrides the original method. In other words, this method is called (which only sets a local variable, not the value in the store) instead of the correct store action method.